### ISSUE TITLE
Error when an issue has been deleted on Google Code 

### ISSUE THREAD

**Issue by ozh** [trentm/googlecode2github#7]
Hi!

Not sure you're still devoting some time to this project, but I figured I could ask for help anyway :)

**TL;DR: I'm trying to post a dummy issue on GH when the Google Code issue has been deleted, but I speak Python as much as I understand Chinese.**

I'm trying to migrate a fairly big project from Google to Github. My problem is that, on Google, I occasionally deleted a few issues (mostly because of spam, or genuine stupidity) to have better search results when searching in past issues.

When your script encounters a missing issue, it skips to the next one, fine, but from now on issue can't have the same number so they are all skipped.

Example: my Google Projet has no Issue 38:

```
Migrating issue 37.
     from: http://code.google.com/p/yourls/issues/detail?id=37
       to: https://github.com/ozh/dummytest/issues/37
Migrating issue 39.
     from: http://code.google.com/p/yourls/issues/detail?id=39
       to: https://github.com/ozh/dummytest/issues/38
  WARNING: github issue id would not match, skipping
```

and so on....

Instead of `forcing` to create an issue with a mismatching number, I'd rather create a dummy issue on Github (something like [Issue deleted])

Would you give a total Python noob some pointers? I tried replacing the  

`print "  WARNING: github issue id would not match, skipping"` 

with a   

```
        title = "Issue Deleted"
        body = "Issue Deleted"
        req = {
            'url': '/repos/%s/issues' % gh_proj,
            'body': {
                "title": title.encode('utf-8'),
                "body": body.encode('utf-8'),
            }
        }
        response, content = _github_api_post(req['url'], req['body'])
        print "  ZOMG with issue %s" % id
        return
```

but all I got was all issues created after the missing one be "Deleted" so I'm obviously missing other parts of the code but, again, that's the very first time I even _read_ Python :)

No sweat if you just don't want to spend time on that project :-)

Cheers

Ozh

**Comment by ozh** [comment]
Nevermind. This frustrated me too much and I just had to write my own script in a language I know :)
Thanks for yours, it made things much easier!

**Comment by ooxi** [comment]
Hi,

Could you publish your Script, too? Perhaps somebody Else will have a similar Problem in the future :)

-- ooxi

Violetland — An open source cross-platform game similar to Crimsonland — http://violetland.github.com

**Comment by ozh** [comment]
Yeah, sure: https://github.com/ozh/google-issues-to-github

It's a PHP CLI script. Already functional as of now, I'll polish a few more thing tomorrow.
